pyro/autohelp/ast_visitor.py

```diff
@@ -0,0 +1,361 @@
+import enum
+from re import S
+from typing import Optional
+
+import libcst
+
+
+class Actions(enum.Enum):
+    # we don't care about any of these values, just using them as sentinels
```

Comment:
We do care about these values now; or at least the combined bits; if you remember the conversation regarding caching based on the actions taken.

Just putting that reminder here